packages/icons/utils/internal.js

```diff
@@ -1,14 +1,11 @@
+// @ts-check
 import { useTheme, Box } from '@mui/material'
 import * as React from 'react'
 import { MaskIconPaletteContext } from './MaskIconPaletteContext.js'
 
-/**
- * @typedef {[currentVariant: null | string[], url: string, jsx: object | null, supportColor?: boolean]} RawIcon
- */
-
 /**
  * @param {string} name
- * @param {Array<RawIcon>} variants
+ * @param {Array<import('./internal.js').__RawIcon__>} variants
```

Comment:
I think the previous structure would be cleaner.

Reply:
This new structure is intended to lazy load and make it smaller. The tuple structure does not allow me to omit some fields.